db/migrate/20160404125552_add_column.rb

```diff
@@ -0,0 +1,5 @@
+class addColumn < ActiveRecord::Migration
+  def change
+  	add_column :employees, :department, :string
+  end
+end
```

Comment:
deleted one, had the same id as previous one so had to create another lol, deleted the duplicate